sdk/core/azure-core/azure/core/pipeline/transport/base.py

```diff
@@ -166,8 +168,8 @@ def set_xml_body(self, data):
             self.data = None
         else:
             bytes_data = ET.tostring(data, encoding="utf8")
-            self.headers['Content-Length'] = str(len(bytes_data))
-            self.data = bytes_data
+            self.data = bytes_data.replace(b"encoding='utf8'", b"encoding='utf-8'")
```

Comment:
Is it always guaranteed to be single quotes around `utf8` ? (asking because I don't actually know)